docs/virtual_machines/disks_and_volumes.md

```diff
@@ -410,7 +403,7 @@ DataVolumes have finished their clone and import phases.
           disks:
           - disk:
               bus: virtio
-            name: volume1
+            name: disk1
```

Comment:
hmm... the volume name seems to be `volume1`, no?

Reply:
not sure what I was thinking here... i unfixed it.